lib/src/components/supa_email_auth.dart

```diff
@@ -74,6 +74,8 @@ class SupaEmailAuth extends StatefulWidget {
 
   final List<MetaDataField>? metadataFields;
 
+  final Map<String, dynamic>? extraMetadata;
```

Comment:
It would be great to add some descriptions here. I noticed that the `metadataFields` doesn't have any descriptions either. Would you be able to add something there as well?
```suggestion
  /// Values to be added as the user's meta data.
  final Map<String, dynamic>? extraMetadata;
```

Reply:
Added doc for both + typo somewhere else. I was on a rush so forgot to do it.
Thanks for the review!